openedx/core/djangoapps/catalog/utils.py

```diff
@@ -109,15 +109,17 @@ def get_program_types(name=None):
         return []
 
 
-def get_programs_with_type(types=None):
+def get_programs_with_type(types=None, include_hidden=True):
```

Comment:
This is a story regarding excluding hidden programs from dashboard (https://openedx.atlassian.net/browse/LEARNER-1140). As per Janaky: "we definitely want hidden courses and programs to show on the Learner Dashboards"